tests/tests/test-fees/test-fee-multiplier.ts

```diff
@@ -425,3 +425,18 @@ describeDevMoonbeam("Fee Multiplier - XCM Executions", (context) => {
     expect(initialValue.eq(postValue), "Fee Multiplier has changed between blocks").to.be.true;
   });
 });
+
+describeDevMoonbeam("TransactionPayment Runtime Queries", (context) => {
+  it("should be able to query length fee", async function () {
+    const adjusted_length_fee = await context.polkadotApi.call.transactionPaymentApi.queryLengthToFee(1n);
+    expect((adjusted_length_fee as any).toBigInt()).to.eq(1_000_000_001n);
```

Comment:
What can I do to avoid needing `as any` here?

Reply:
Updates the typescript api to match
```
cd typescript-api
npm ci
cd ../tests
npm run setup-typescript-api
```

Reply:
@notlesh you dont need to build any new types or api-augment pkgs, that's been done already.

All you need todo is update the api-augment package to 2301 which contains the updated interfaces (and removes the need for `any` locally for me at least)

in: `moonbeam/tests`
```
npm i @moonbeam-network/api-augment@latest 
```